packs/hpe-icsp/actions/icsp_ca_get.yaml

```diff
@@ -0,0 +1,13 @@
+---
+  name: "icsp_ca_get"
+  runner_type: "run-python"
+  description: "Retreive CA Certificate from ICSP"
```

Comment:
I'll admit this was just a additional action I created when I was trying to deal with CA's to enable ssl. i needed a way to pull the ICSP server cert so it's just returns the contents of the get request.